code_gen/README.md

```diff
@@ -0,0 +1,34 @@
+# AI Code Gen for Pipedream Components
+
+Generate components (currently actions only) using OpenAI GPT-4.
+
+
+### Run
+
+```
+APP=slack
+PROMPT="how to send myself a direct message?"
+python run.py --app "$APP" "$PROMPT"
```

Comment:
Check out https://python-poetry.org/docs/pyproject/#scripts — you can define a script that can be installed to the local environment when a user runs `poetry install`.

For example, if you put this in your `pyproject.toml`:

```
[tool.poetry.scripts]
my-script = 'mypackage.mymodule:main'
```

and run `poetry install`, then `my-script` is available in your shell.

Reply:
I think that might be nice so people can always just run the script in any dir when they're hacking on a component

Reply:
looks awesome!